packages/framer-motion/src/animation/animators/js/index.ts

```diff
@@ -158,7 +158,9 @@ export function animateValue<V = number>({
         }
 
         // Rebase on delay
-        time = Math.max(time - delay, 0)
+        const timeWithDelay = time - delay
```

Comment:
If it is `time - delay`, I would expect this constant to be called `timeWithoutDelay`

Reply:
Shouldn't `time` be called `currentTime`?